jerry-ext/args/args-internal.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef JERRYX_ARGS_INTERNAL_H
+#define JERRYX_ARGS_INTERNAL_H
+
+#include "jerryscript.h"
+
+/**
+ * The iterator structor for JS arguments.
+ */
+struct jerryx_arg_js_iterator_t
```

Comment:
Sorry I don't understand. the other components of this library can access to the internal iterator as long as they include this header `args-internal.h`